test/server/server_fuzz_test.cc

```diff
@@ -58,7 +58,7 @@ makeHermeticPathsAndPorts(Fuzz::PerTestEnvironment& test_env,
       // Tracked at https://github.com/envoyproxy/envoy/issues/9513.
       health_check.mutable_http_health_check()->clear_codec_client_type();
     }
-    for (auto& host : *cluster.mutable_hosts()) {
+    for (auto& host : *cluster.mutable_hidden_envoy_deprecated_hosts()) {
```

Comment:
Should this be updated?

Reply:
Yes, but this should apply to both deprecated hosts and load assignment, not sure why we haven't seen port issues with the latter, probably some false positive issues open as a result, will fix.